src/app/modules/game/pages/game/game.component.html

```diff
@@ -1,15 +1,16 @@
 
 <div class="game">
-
-  <div class="game__room-code">
-    <app-room-code></app-room-code>
-  </div>
-
-
-  <div class="game__column">
-    <button mat-icon-button (click)="toggleSidenav('leftSidenav')" (click)="toggleDisplay('leftArrowShow')" *ngIf="leftArrowShow"
-      matTooltip="Show filters" color="accent">
-      <mat-icon aria-hidden="false">keyboard_arrow_right</mat-icon>
+    <div class="game__column">
+    <div class="game__room-code">
+      <app-room-code></app-room-code>
+    </div>
+    <button mat-icon-button class="hide-button left" (click)="toggleSidenav('leftSidenav')" (click)="toggleDisplay('leftArrowShow')" *ngIf="!leftArrowShow"
+    >
+    <mat-icon class="icon-arrow" ></mat-icon>
+    </button>
+    <button mat-icon-button class="show-button left" (click)="toggleSidenav('leftSidenav')" (click)="toggleDisplay('leftArrowShow')" *ngIf="leftArrowShow"
```

Comment:
Maybe you can use [ngClass] that will add "left" class to button on condition. Therefore, you won't have 2 equal buttons. And you can change mat-icon with .show-button--left > .icon-arrow instead of adding "left" to mat-icon too.